certora/harness/MorphoHarness.sol

```diff
@@ -10,16 +10,16 @@ contract MorphoHarness is Morpho {
 
     constructor(address newOwner) Morpho(newOwner) {}
 
-    function wad() external pure returns (uint256) {
-        return WAD;
+    function toMarketParams(Id id) external view returns (MarketParams memory) {
```

Comment:
```suggestion
    function idToMarketParams_(Id id) external view returns (MarketParams memory) {
```

Reply:
What do you guys think handling naming/maitainance discussions in a separate PR as this may break other specifications requiring yet more changes. Again the priority here is that this PR is blocking pre-liquidation and this naming suggestion is not immediately related.

Reply:
it's easy changes no?

Reply:
Implemented in https://github.com/morpho-org/morpho-blue/pull/700/commits/662d48c6cc31913ceac28b231cf07354402263f9. Easy was not the concern here, but it's done now. I hope it doesn't break anything Metamorpho…🤞